### DISCUSSION TITLE
Cannot Login to Download Likes

### DISCUSSION THREAD

**Discussion by cheese529** [TumblThreeApp/TumblThree#322] [Q&A (Question & Answer)]
Windows 10 Home
Latest Version of Google Chrome and TumblThree

Hi there, just downloaded this program to download my own tumblr likes and private blogs I have created. As soon as I try to authenticate my tumblr account in the connections tab I get this error after clicking it. Then it gives me no other option but to exit my application. Tried everything from clearing cookies to running the application as admin to reinstalling it as well. 
![image](https://user-images.githubusercontent.com/94316708/141694637-56a6f7f2-66df-4d55-a4cf-ca3caeb51119.png)

**Comment by thomas694** [comment]
Hello,
before starting TumblThree again, please make sure that no processes with "TumblThree" or "CefSharp" in the name are still running in the Task Manager. You can also take a look into the TumblThree.log (%LOCALAPPDATA%\TumblThree) and debug.log in the installation folder to see if there are any errors/hints.

**Comment by thomas694** [comment]
@ all
If you experience this crash right before, while doing the login or after closing the dialog, then on your system the "[Visual C++ Redistributable 2015-2019](https://support.microsoft.com/en-us/topic/the-latest-supported-visual-c-downloads-2647da03-1eea-4433-9aff-95f26a218cc0)" ([x86](https://aka.ms/vs/16/release/vc_redist.x86.exe) or [x64](https://aka.ms/vs/16/release/vc_redist.x64.exe)) is missing. You can download it from MS.
Download and install the required package, then start TumblThree again and try to log in.

**Comment by thomas694** [comment]
The issue has been closed. You can still comment. Feel free to ask for reopening the issue if needed.
